### ISSUE TITLE
Quantity of aspect`s contacts in aspect navigation panel changes only after clicking on aspect navigation panel or after refreshing

### ISSUE THREAD

**Issue by juliaviluhina** [diaspora/diaspora#6445]
It seems like #6424

Steps
1. Sign in Diaspora account
2. In user menu on header click Contacts (/contacts)
3. In aspect navigation panel (id="aspect_nav") select aspect which is contain some contacts
4. Change aspects in some contact, which is shown on right side, using "Remove contact" or "Add contact" buttons

Expected result
In aspect navigation panel information about quantity of contacts should change afler removing or adding contact

Actual result
In aspect navigation panel information about quantity of contacts does not change afler afler removing or adding contact. (However, any clicking on aspect navigation panel gives expected result)

**Comment by jhass** [comment]
Duplicate of #6424 / #2900 and also solved by #6240
